activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+*   Add support for `rails console --sandbox` for multiple database applications.
+
+    Multiple database applications will now be rolled back on exit.
```

Comment:
I don't know if this changelog is really necessary. It implies it's adding support for multiple databases but it's really fixing a bug. We don't add changelogs when fixing a bug. Can you remove this change?

Reply:
done 👍